hvplot/converter.py

```diff
@@ -381,7 +387,7 @@ def __init__(
         self, data, x, y, kind=None, by=None, use_index=True,
         group_label=None, value_label='value', backlog=1000,
         persist=False, use_dask=False, crs=None, fields={},
-        groupby=None, dynamic=True, grid=None, legend=None, rot=None,
+        groupby=None, dynamic=True, grid=None, legend=None, robust=None, rot=None,
```

Comment:
Perhaps let's stuff it after `clim`? Trying to have some sort of semantic ordering in that mess :D